examples/config/config

```diff
@@ -516,7 +517,7 @@ set ssl_policy fail
 
 # === Post-load misc commands ================================================
 spawn_sync_exec @scripts_dir/load_cookies.sh
-spawn_sync_exec @scripts_dir/load_cookies.sh @data_home/session-cookies.txt
+spawn_sync_exec @scripts_dir/load_cookies.sh @(echo "${UZBL_SESSION_COOKIE_FILE}")@
```

Comment:
On Wed, Feb 11, 2015 at 08:43:39 -0800, Samuel Loury wrote:
> It suppose the script examples/data/scripts/util/uzbl-dir.sh has been
> sourced somewhere.

The uzbl-dir.sh code is meant for use inside of the scripts; this is the
configuration file which can't assume its use (the user would need to
run it for it to take effect). The configuration file is also supposed
to work with uzbl-browser, uzbl-tabbed, or any other tool when uses
uzbl-core directly.

> The same assumption is made at the beginning of this file. The setting relying on
> prefix, data_home, data_dirs, cache_home and config_home don't work either if
> the XDG environment variables are not set.
> 
> For instance, the ctrl-m binding (adding a bookmark) won't work if
> data_home is not correctly set, which is the case if XDG_DATA_HOME is
> not correctly set, which could be the case if
> examples/data/scripts/util/uzbl-dir.sh is not sourced.

The `_HOME` variables are handled in multiple places. First,
uzbl-browser sets them up for itself and uzbl-core. uzbl-core will also
ensure that the `_HOME` variables exist and will use the `_DIRS`
variants for finding files as a fallback, but it won't set those.